src/helper/helper.js

```diff
@@ -300,6 +340,8 @@ class DataLayerHelper {
 }
 
 window['DataLayerHelper'] = DataLayerHelper;
+DataLayerHelper.prototype['process'] = DataLayerHelper.prototype.process;
+DataLayerHelper.prototype['registerProcessor'] = DataLayerHelper.prototype.registerProcessor;
```

Comment:
As an alternative, have you tried annotating these methods with @export?

https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#export-export-sometype

Reply:
Whoa I wish I had known this before, #52 got merged a couple seconds after I read this comment, and that PR also does it this way, could you update that here? This method does work, just add the line `generate_exports: true,` to the gruntfile

Reply:
I'll update it accordingly 👍